albumentations/core/composition.py

```diff
@@ -271,6 +273,13 @@ def _check_args(self, **kwargs) -> None:
             if internal_data_name in check_bbox_param and self.processors.get("bboxes") is None:
                 raise ValueError("bbox_params must be specified for bbox transformations")
 
+    def _make_targets_contiguous(self, data):
+        targets = ["image", "mask"]
+        for target in targets:
+            if target in data and isinstance(data[target], np.ndarray) and not data[target].flags["C_CONTIGUOUS"]:
+                data[target] = np.ascontiguousarray(data[target])
+        return data
```

Comment:
Thank you for your contribution.
I think it is better to write a generalized function that will covert all np.ndarrays to a contiguous format.
```python
@staticmethod
def _make_targets_contiguous(data: typing.Dict[str, typing.Any]) -> typing.Dict[str, typing.Any]:
    result = {}

    for key, value in data.items():
        if isinstance(value, np.ndarray):
            value = np.ascontiguousarray(value)
        result[key] = value

    return result
```

Also, we do not need this check `data[target].flags["C_CONTIGUOUS"]` because if the array is contiguous `np.ascontiguousarray` will return it unchanged and will not create a copy.

Reply:
Yes. you're right! I will fix it.